### ISSUE TITLE
Collection has no remove button

### ISSUE THREAD

**Issue by zachzundel** [SynBioHub/synbiohub#466]
There is no remove button on the Collection page, and in our meeting, @cjmyers and I couldn't remember why. If we can't remember why in a week or so, I will convert this issue to adding the button.

**Comment by cjmyers** [comment]
The Remove buttons on the View pages and the Manage Submissions pages are subtlety different.

1) The Remove button on a view page removes the TopLevel object being viewed and any child objects (i.e., remove endpoint).
2) The Remove button on the Manage Submissions page removes all objects that share the URI prefix with the Collection being removed (i.e., removeSubmission endpoint).

The later functionality is there to enable the removal of an entire submission.  This is important since if we just removed the Collection, we would end up with a bunch of hidden TopLevels that could not be easily found again, since they are not a member of any Collection.

For this reason, I think if we add a button to Collection for Remove it should be removeSubmission.  My guess is that the Remove button is not there due to this mismatch in semantics, but I think it might be okay to include it as long as we are careful to link it to the correct endpoint.

**Comment by zachzundel** [comment]
I agree. I do think there should be an alert when you try to remove a collection, something along the lines of, 'This will also delete all X members of this collection' so that it is clear what is happening.

**Comment by cjmyers** [comment]
Note it only deletes members if they share uri prefix with the collection since this implies they were submitted with the collection.

Chris

**Comment by cjmyers** [comment]
This one is problematic, since removeSubmission will remove all in this submission even if the Collection is just a subset of the submission. Should probably modify the remove method for collection to just remove just the Collection, perhaps with the option to remove the members too.

**Comment by cjmyers** [comment]
Modified remove on collection to remove only the collection members that share the same URI prefix as the collection being removed.
